python_modules/libraries/dagster-azure/dagster_azure_tests/adls2_tests/test_io_manager.py

```diff
@@ -74,6 +74,7 @@ def basic_external_plan_execution():
 
 
 @pytest.mark.nettest
+@pytest.mark.skip
```

Comment:
nit: github issue link (unless you are confident you can get it back on very soon)